src/c++/perf_analyzer/genai-pa/genai_pa/llm_metrics.py

```diff
@@ -152,38 +173,81 @@ def __repr__(self):
         attr_strs = ",".join([f"{k}={v}" for k, v in self.__dict__.items()])
         return f"Statistics({attr_strs})"
 
+    def _is_throughput_field(self, field: str):
+        return field in Metrics.throughput_fields
+
     def _is_time_field(self, field: str):
-        time_fields = [
-            "inter_token_latency",
-            "time_to_first_token",
-            "end_to_end_latency",
-        ]
-        return field in time_fields
+        return field in Metrics.time_fields
 
     def pretty_print(self):
         table = Table(title="PA LLM Metrics")
 
         table.add_column("Statistic", justify="right", style="cyan", no_wrap=True)
-        stats = ["avg", "min", "max", "p99", "p95", "p90", "p75", "p50", "p25"]
+        stats = ["avg", "min", "max", "p99", "p90", "p75"]
```

Comment:
are we excluding p25, p50, and p95?

Reply:
Yes, we're only including those in the CSV. The printed report shows the highlights, the CSV will show the details (all of the percentiles we calculate). The screenshot is in the pull request description for reference.